docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,264 @@
+```
+
+### Linux
+
+On Linux, Ionide also uses Mono.  If you're on Debian or Linux, you can use the following:
+
+```
+sudo apt-get update
+sudo apt-get install mono-complete fsharp
+```
+
+## Installing Visual Studio Code and the Ionide plugin
+
+You can install Visual Studio Code on the [code.visualstudio.com](https://code.visualstudio.com) website.  After that, there are a two ways to find the Ionide plugin:
+
+1. Use the Command Palette (ctrl-P on Windows, ⌘-P on macOS, ctrl-shift-P on Linux) and type the following:
+
+    ```
+    ext install Ionide
+    ```
+
+2. Click the Extensions icon and search for "Ionide":
+
+    ![](media/getting-started-vscode/vscode-ext.png)
+
+The only plugin required is **Ionide-fsharp**.  However, you can also install **Ionide-FAKE** and **Ionide-Paket** to get [FAKE](http://fsharp.github.io/FAKE/) and [Paket](https://fsprojects.github.io/Paket/) support.
+
+## Creating your first project Ionide
+
+To create a new F# project, open the Command Palette (ctrl-P on Windows, ⌘-P on macOS, ctrl-shift-P on Linux) and type the following:
+
+```
+>f#: New Project
+```
+
+You should see something like this:
+
+![](media/getting-started-vscode/vscode-new-proj.png)
+
+Select it by hitting **Enter**, which will take you to this step:
+
+![](media/getting-started-vscode/vscode-proj-type.png)
+
+This will select a template for a specific type of project.  There are quite a few options here, such as an FsLab template for Data Science or Suave template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach this step:
+
+![](media/getting-started-vscode/vscode-new-dir.png)
+
+Here you can choose to create the project in a specific directory.  If you leave it blank, it will choose the current directory Visual Studio Code is opened in.  This article uses the current directory.  Once you've made your decision, hit **Enter** to bring up the final stage:
+
+![](media/getting-started-vscode/vscode-proj-name.png)
+
+This lets you name your project.  F# uses PascalCase for project names.  This article uses `ClassLibrarySample` as the name.  Once you've entered the name you want for your project, hit **Enter**.
+
+If you followed these steps, you should get the Explorer on the left-hand side to look something like this:
+
+![](media/getting-started-vscode/vscode-new-proj-explorer.png)
+
+This template generates a few things you'll find useful:
+
+1. The F# project itself, underneath the `ClassLibraryDemo` folder.
+2. The correct directory structure for adding packages via [`Paket`](http://fsprojects.github.io/Paket/).
+3. A cross-platform build script with [`FAKE`](http://fsharp.github.io/FAKE/).
+4. The `paket.exe` executable which can fetch packages and resolve dependencies for you.
+5. A useful `.gitignore` file if you wish to add this project to Git-based source control.
+
+## Writing some code
+
+Open the `ClassLibraryDemo` folder.  You'll notice a few files:
```

Comment:
We put in our guidelines that paths and filenames should be italic since the ASP.NET team has been using that.

Also, instead of "You'll notice a few files", wouldn't be better to say "You should see the following files:"